app/controllers/publishers/vacancies_controller.rb

```diff
@@ -59,6 +59,13 @@ def summary
     @feedback_form = Publishers::JobListing::FeedbackForm.new
   end
 
+  def applications
+    @vacancy = VacancyPresenter.new(@vacancy)
+    @sort = Publishers::VacancySort.new(current_organisation, @selected_type).update(column: params[:sort_column])
```

Comment:
i realise this sort isnt the right sort, its just to see the select in UI for now